scripts/utils/cli/params.js

```diff
@@ -124,12 +129,34 @@ module.exports.addNewValue = (yargs, defaultParam = NEW_VALUE.default) => {
     yargs,
     NEW_VALUE.name,
     NEW_VALUE.alias,
-    "number",
+    "string",
     `New value to se in the settings. By default: ${defaultParam}`,
     defaultParam
   );
 };
 
+module.exports.addMaxValue = (yargs, defaultParam = MAX_VALUE.default) => {
+  newOption(
+    yargs,
+    MAX_VALUE.name,
+    MAX_VALUE.alias,
+    "string",
+    `Min value to set in the settings. By default: ${defaultParam}`,
```

Comment:
We should update     `Min value to set in the settings. By default: ${defaultParam}`,
with     `Max value to set in the settings. By default: ${defaultParam}`,